build.gradle

```diff
@@ -36,7 +36,7 @@ jar {
     baseName = 'eiffel-remrem-semantics'
     // If this version is updated please update it in Event.java 
     // com.ericsson.eiffel.remrem.semantics.events
```

Comment:
I understand that version is no longer used in meta data for event generation. Comment above must be removed.